src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -34,7 +39,7 @@ then
 	echo "Done in $ELAPSED_TIME sec(s)."
 
 	echo "Running pip install..."
-	pip install --prefer-binary -r requirements.txt | ts $TS_FMT
+	pip install --cache-dir $PIP_CACHE_DIR --prefer-binary -r requirements.txt | ts $TS_FMT
```

Comment:
How is the performance of a second build? do you see a similar experience as before this change?

Reply:
locally I didn't see any difference (i.e. running integration tests etc on local containers)